Validation/RecoTrack/python/TrackValidation_cff.py

```diff
@@ -716,6 +716,7 @@ def _uniqueFirstLayers(layerList):
 tpClusterProducerPixelTrackingOnly = tpClusterProducer.clone()
 # Need to use the modifier to customize because the exact EDProducer type depends on the modifier
 gpu.toModify(tpClusterProducerPixelTrackingOnly, src = "tpClusterProducerHeterogeneousPixelTrackingOnly")
+(~gpu).toModify(tpClusterProducerPixelTrackingOnly, pixelClusterSrc = "siPixelClustersPreSplitting")
```

Comment:
mhm, this is above the level of ugliness that I would like to have...

can we set it to `siPixelClustersPreSplitting` by default, and change it to something else for the `gpu` case ?